python/tests/webdriver_test.py

```diff
@@ -0,0 +1,21 @@
+import pytest
```

Comment:
Must be in `python/test_sample/`. In this package we have only unit tests.
The same about `python/tests/unit_test.py` file.